tasks/system_probe.py

```diff
@@ -669,6 +669,85 @@ def test(
         raise Exit(code=1, message="system-probe tests failed")
 
 
+@task(
+    help={
+        "package": "The package to test. REQUIRED ",
+        "skip_object_files": "Skip rebuilding the object files.",
+        "run": "The name of the test to run. REQUIRED",
+    }
+)
+def test_debug(
+    ctx,
+    package,
+    bundle_ebpf=False,
+    runtime_compiled=False,
+    co_re=False,
+    skip_linters=False,
```

Comment:
We can default this to true for this task.

Reply:
I think you can remove it and the call the the linters altogether for this task.

Reply:
I agree running linters is not useful for this task. The reason I kept it is to keep the same API than the `test` task.

Reply:
Fixed.